app/src/main/java/com/sun_asterisk/moviedb_50/screen/listmovie/ListMovieFragment.kt

```diff
@@ -19,27 +41,145 @@ class ListMovieFragment : Fragment(), ListMovieContract.View {
                 MovieLocalDataSource
             )
         presenter = ListMoviePresenter(movieRepository)
+    }
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        return inflater.inflate(R.layout.fragment_list_movie, container, false)
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        initToolBar()
         initPresenter()
+        initAdapter()
     }
 
     override fun onGetMoviesSuccess(movies: List<Movie>) {
-        TODO("Not yet implemented")
+        if (adapter.itemCount == 0) {
+            adapter.updateData(movies)
+        } else {
+            adapter.insertData(movies)
+        }
```

Comment:
```suggestion
            adapter.insertData(movies)
```
Thử dùng mỗi cái này được k nha.